XMonad/Hooks/DynamicLog.hs

```diff
@@ -67,33 +74,35 @@ module XMonad.Hooks.DynamicLog (
 
 -- Useful imports
 
-import Codec.Binary.UTF8.String (encodeString)
```

Comment:
I don't think a pr like this should change all of the import formatting; if this is desirable it can be done in a separate pr

Reply:
that was stylish haskell